### DISCUSSION TITLE
Is it possible to import a spice netlist into Skidl?

### DISCUSSION THREAD

**Discussion by tapegoji** [xesscorp/skidl#214] [Q&A]
I have created a SPICE netlist using SKiDL and saved it. Now I want to import it into another script. Is that possible?

SKiDL can import SKiDL netlists without any problem. However, I am not sure if it can import SPICE netlists.

**Comment by devbisme** [comment]
I wrote a utility called `netlist_to_skidl.py` that converts a KiCad netlist into a SKiDL program. I suppose you could do something similar for SPICE netlists.

But why? If you already have a SKiDL script that generates a SPICE netlist, why not import that into your master script and avoid the intermediate conversion?

**Reply by tapegoji** [reply]
I do plan to have it imported at some point. for some testing, I thought it's easier to do the conversion.
Thank you for the prompt reply.
